### PR TITLE
Rule/ddb backup

### PR DESCRIPTION
**Pull request by vincentzan**
add rule for ddb backup config

### CODE REVIEW

File: packages/core/src/clients/dynamodbClient.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { DynamoDBClient } from '@aws-sdk/client-dynamodb'; // ES Modules import
```
Comment (adelego) [inline packages/core/src/clients/dynamodbClient.ts:1]: @vincentzan you and Anaïs are stepping on each others toes. You both have created an DDB client

File: packages/core/src/rules/dynamodbBackupConfig/index.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import fetchBackupProtectedResourceArns from '../../aws-sdk-helpers/backup/fetchBackupProtectedResourceArns';
+import { Rule } from '../../types';
+
+const run: Rule['run'] = async resourceArns => {
+  const backupProtectedResourcesArn = await fetchBackupProtectedResourceArns();
+  console.log(backupProtectedResourcesArn);
```
Comment (adelego) [inline packages/core/src/rules/dynamodbBackupConfig/index.ts:6]: You forgot to clean up your logs

File: packages/core/src/rules/dynamodbBackupConfig/doc.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+---
+
+## Suggested Actions:
+
+- Create a BackupPlan in the Backup Service;
+- Configure the name of the Backup plan and the Backup Rules. These include when you run your Backup jobs, in what storage class you want to keep the Backups, in which AWS region you want them saved and your how long you retain them. If applicable to your use case, you might want to use the point in time recovery which backs up your table data automatically with per-second granularity
+- After creating the plan, you can simply assign resources to this plan using the resource name. For the scope of this rule, you want to add the name of the DynamoDB table to back up.
+
+## Links :
```
Comment (adelego) [inline packages/core/src/rules/dynamodbBackupConfig/doc.md:14]: Can you add links to sls framework and cdk on how to add backups?

File: packages/core/src/rules/dynamodbBackupConfig/doc.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# DynamoDB: back up your tables regularly
+
+DynamoDB is a NoSQL database service and as such it is widely used in any software application. It is recommended that you keep a secondary copy of the data stored in your database, in case of a primary data failure (e.g. introduction of corrupted data, handling mistakes etc.).
+AWS Backup Service allows you to define Backup plans. These are periodically updated snapshots of the database, the state of which you can roll back your table to.
+
+---
+
+## Suggested Actions:
+
+- Create a BackupPlan in the Backup Service;
```
Comment (adelego) [inline packages/core/src/rules/dynamodbBackupConfig/doc.md:10]:
```[markdown]
- In the AWS console, in the DynamoDB service, create a BackupPlan in the Backup Service;
```

**Review by adelego — APPROVED** [review]

**Comment by PChol22** [comment]
Fixes #191

**Pull request merged**